### ISSUE TITLE
[SIP] Initial Validator Deposits

### ISSUE THREAD

**Issue by steao** [SharedStake/SharedStake-ui#17]
TL;DR: 17 validators are currently on ETH2. Should we wait for additional testing or send all available validators to the ETH2 contract?

Summary: We have deposited the first batch of 17 validators to the ETH2 deposit contract. We have almost 100 additional validators worth of ETH in our contract that are ready for migration. Should we deposit the additional 100 into the ETH2 deposit contract or should we wait until the first batch of validators are up and running and the infrastructure has been fully tested?

Discussion: https://discord.com/channels/779046626236563487/811039103066963985 

Note: Current validator deposits require mandatory 14-day onboarding before validating blocks.
https://beaconscan.com/validator/0xab48faee02350fad44b52198214b4abaa93598b9c9526e0fee746f03995102036b76d3374907b6d21890a1895dedadaf

**Comment by steao** [comment]
Addressed and updated on the [snapshot.page](https://snapshot.page/#/sharedstake.eth/proposal/QmVKySHU4J5iUdfpq7o5qAjrHscmi4q3k4SGCPE5KGvYLc)
